akka-docs/src/main/paradox/typed/persistence.md

```diff
@@ -207,8 +207,8 @@ Scala
 Java
 :  @@snip [BasicPersistentBehaviorTest.java](/akka-persistence-typed/src/test/java/jdocs/akka/persistence/typed/BasicPersistentBehaviorTest.java) { #event-handler }
 
-@scala[These are used to create an `EventSourcedBehavior`:]
-@java[These are defined in an `EventSourcedBehavior`:]
+@scala[These are used to create an @scaladoc[EventSourcedBehavior](akka.persistence.typed.scaladsl.EventSourcedBehavior):]
+@java[These are defined in an @javadoc[EventSourcedBehavior](akka.persistence.typed.javadsl.EventSourcedBehavior):]
```

Comment:
There is class `Effect` in the file. Which link should I use: `akka.persistence.typed.state.javadsl.Effect` or `akka.persistence.typed.javadsl.Effect`? I don't know :(

Reply:
`akka.persistence.typed.state.javadsl.Effect` is used for the Durable State feature (https://doc.akka.io/docs/akka/current/typed/index-persistence-durable-state.html). On this page the correct effect type is `akka.persistence.typed.javadsl.Effect`.